### PR TITLE
[SYCL][UR] Do not build tests when downloading unified-runtime

### PR DESCRIPTION
**Pull request by jandres742**

### CODE REVIEW

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -23,6 +23,8 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-suggest-override")
   endif()
 
+  SET(UR_BUILD_TESTS "0" CACHE STRING "0")
```
Comment (smaslov-intel) [inline sycl/plugins/unified_runtime/CMakeLists.txt:26]: This deserves a comment

**Review by smaslov-intel — APPROVED** [review]

File: sycl/plugins/unified_runtime/CMakeLists.txt (hunk 1 above)
Comment (smaslov-intel) [inline sycl/plugins/unified_runtime/CMakeLists.txt:26]:
```suggestion
  set(UR_BUILD_TESTS "0" CACHE STRING "0")
```

Comment (jandres742) [reply sycl/plugins/unified_runtime/CMakeLists.txt]: thanks @smaslov-intel . Fixed.

**Review by smaslov-intel — APPROVED** [review]

File: sycl/plugins/unified_runtime/CMakeLists.txt (hunk 1 above)
Comment (jandres742) [reply sycl/plugins/unified_runtime/CMakeLists.txt]: @smaslov-intel , yes, added.

**Pull request merged**